example/imports.yaml

```diff
@@ -1,32 +1,108 @@
+cluster: |
+  apiVersion: v1
+  kind: Config
+# ... <please insert your kubeconfig here>
+
 hostingCluster:
-  kubeconfig: |
-    apiVersion: v1
-    kind: Config
-#   ... <please insert your kubeconfig here>
-  namespace: garden-123
+  namespace: garden-4
   infrastructureProvider: gcp # aws|gcp|alicloud
 
 virtualGarden:
+  deleteNamespace: true
+
   etcd:
     storageClassName: my-virtual-garden-storage-class
     backup:
       infrastructureProvider: gcp # aws|gcp|alicloud
-      region: europe-west5
+      region: europe-west555
```

Comment:
?

Reply:
Just to be sure not to deploy something working. I have reverted to the origin.